ArchiSteamFarm/Plugins/PluginsCore.cs

```diff
@@ -871,7 +873,7 @@ public static class PluginsCore {
 
 				await plugin.OnPluginUpdateProceeding().ConfigureAwait(false);
 
-				if (!Utilities.UpdateFromArchive(zipArchive, assemblyDirectory)) {
+				if (!await Utilities.UpdateFromArchive(zipArchive, assemblyDirectory).ConfigureAwait(false)) {
```

Comment:
If I remember correctly, the issue was created because of my suggestion to load zipArchive directly from response stream, without loading it to memory first. Is it still possible?

Reply:
> Potential improvement for the future (totally outside of scope of this PR) will be to start extracting of zip file as the stream is downloaded. But this can wait, firstly we need to get new update procedure right.